jql-core/src/main/java/io/github/benas/jql/core/ConstructorIndexer.java

```diff
@@ -43,11 +43,10 @@ public ConstructorIndexer(MethodDao methodDao, ParameterIndexer parameterIndexer
     }
 
     public void index(ConstructorDeclaration constructorDeclaration, int typeId) {
-        int modifiers = constructorDeclaration.getModifiers();
         List<Parameter> parameters = constructorDeclaration.getParameters();
-        String name = constructorDeclaration.getName();
+        String name = constructorDeclaration.getNameAsString();
```

Comment:
Names are now nodes instead of strings, so there's a helper method to get the string value.

Reply:
ok, got it